en/flight_modes_fw/takeoff.md

```diff
@@ -0,0 +1,127 @@
+# Takeoff Mode (Fixed-wing)
+
+[<img src="../../assets/site/position_fixed.svg" title="Position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)

```

Comment:
Restructured but content largely the same. The main difference is that this doesn't really refer to missions much at all.